tools/triage/add_triage_signature.py

```diff
@@ -2362,6 +2362,44 @@ def _process_ticket(self, url, issue_key):
             self._update_triaging_ticket("Controller logs contain some warnings:\n" + f"{{code}}{warning_text}{{code}}")
 
 
+class UserHasLoggedIntoCluster(Signature):
+    """
+    This signature will detect customer login to the cluster.
+    """
+
+    USER_LOGIN_PATTERN = re.compile(r"Accepted publickey for core from")
```

Comment:
This will only detect SSH logins. Can we detect password-based logins as well? Users can [set a password](https://github.com/openshift/assisted-service/blob/master/docs/set-discovery-password.md) on discovery ISOs

Reply:
After looking into this, I can see that whenever a manual login takes place, the phrase 

"...session opened for core by...."

Will be present irrespective of the authentication method and it looks like the journal logs usually show sessions opened for the root user during installation.

For this reason I think this is a fairly reasonable heuristic

WDYT?

Reply:
If it catches both methods, I think it's good
we can even change it to ``re.compile(r"session opened for user .+ by")`` to make it future-proof for username changes

Reply:
agreed, came here to write this (more or less)

Reply:
PR updated to use this regex